src/common/constants/cssScale.ts

```diff
@@ -0,0 +1,14 @@
+export const cssScale = {
+  c0: "0",
+  c1: "4px",
+  c2: "8px",
+  c3: "12px",
+  c4: "16px",
+  c5: "20px",
+  c6: "24px",
+  c7: "28px",
+  c8: "32px",
+  c9: "36px",
+  c10: "40px",
+  c11: "44px",
+};
```

Comment:
This works for now. For future projects you can reference: https://www.lightningdesignsystem.com/design-tokens/#category-spacing